packages/terra-time-input/src/TimeInput.jsx

```diff
@@ -621,7 +525,7 @@ class TimeInput extends React.Component {
     const previousStateValue = stateValue;
     const variant = TimeUtil.getVariantFromLocale(this.props);
 
-    if (event.keyCode === KeyCode.KEY_UP) {
+    if (event.keyCode === KeyCode.KEY_UP || (event.key === '=' || event.key === '+')) {
```

Comment:
Not able to find any test written for new hot keys added !  if not created already, We need to create one for each keys.

Reply:
Hotkeys test has been updated - https://github.com/cerner/terra-framework/pull/1966/commits/266e12a9c438f4eba1aa53b6af09614d0f72994a

Reply:
Already existing +/- functionality has =/_ check as well, that's why i have included.
https://github.com/cerner/terra-framework/blob/main/packages/terra-time-input/src/TimeInput.jsx#L485
https://github.com/cerner/terra-framework/blob/main/packages/terra-time-input/src/TimeInput.jsx#L533